man/docker-daemon.8.md

```diff
@@ -197,6 +198,9 @@ unix://[/path/to/socket] to use.
 **-p**, **--pidfile**=""
   Path to use for daemon PID file. Default is `/var/run/docker.pid`
 
+**--raw-logs**=*false*
+  When set to `true`, this the daemon will always output logs in full timestamp format without ANSI coloring. When set to `false` (the default), the daemon will try to output condensed, colorized logs when it detects that it is running inside a terminal.
```

Comment:
Perhaps something like:

```Markdown
Output daemon logs in full timestamp format without ANSI coloring. If this flag is not set,
the daemon outputs condensed, colorized logs if a terminal is detected, or full ("raw")
output otherwise.
```